spec/requests/unwithdraw_spec.rb

```diff
@@ -12,7 +12,9 @@
       post unwithdraw_path(withdrawn_edition.document)
       follow_redirect!
 
-      expect(response.body).to include(I18n.t!("documents.history.entry_types.unwithdrawn"))
+      within("#app-c-metadata") do
+        expect(response.body).to include(I18n.t!("user_facing_states.published"))
+      end
```

Comment:
within doesn't work outside of a capybara session so these assertion in here will never be called.

I'd probably change this request test to just assert that it redirects to document summary and not bother following redirect or asserting or contents since the action isn't setting those.

Reply:
Ah interesting didn't know that, I've followed your advise!